docs/troubleshooting.md

```diff
@@ -0,0 +1,216 @@
+[see its logs to understand what is happening](#how-can-i-see-logs-of-the-running-components)
+
+When one of the required components fails, it uses to print an error in the UI,
+
+e.g. `lost connection to mysql server during query` while running a query might
+mean that `gitbase` went down. 
+
+e.g. `unable to establish a connection with the bblfsh server: deadline exceeded`
+in SQL Lab might mean that `bblfsh` went down.
+
+If the failing component is not successfully restarted in a few seconds, or if it
+goes down when running certain queries, it could be a good idea to [open an issue](https://github.com/src-d/sourced-ce/issues)
+describing the problem.
+
+
+## How Can I See Logs of The Running Components?
+
+```shell
+$ sourced logs [-f] [components...]
+```
+
+Adding `-f` will keep the connection opened, and the logs will appear as they
+come instead of exiting after the last logged one.
+
+You can pass a space-separated list of component names to see only their logs
+(i.e. `sourced-ui`, `gitbase`, `bblfsh`, `gitcollector`, `ghsync`, `metadatadb`, `postgres`, `redis`).
+If you do not pass any component name, there will appear the logs of all of them.
+
+Currently, there is no way to filter by error level, so you could try with `grep`,
+e.g. 
+
+```shell
+sourced logs gitcollector | grep error
+```
+
+will output only log lines where `error` word appears.
+
+
+## How Can I See What Happened with the Scrapers?
+
+_When **souece{d} CE** is initialized with `sourced init local`, the scrapers are
+not relevant because the repositories to analyze comes from your local data, so
+`ghsync` and `gitcollector` status is not relevant in this case._
+
+When running **souece{d} CE** to analyze data from a list of GitHub organizations,
+`gitcollector` component is in charge of fetching  GitHub repositories and `ghsync`
+component is in charge of fetching GitHub metadata (issues, pull requests...)
+
+Once the UI is opened, you can see the progress of the importation in the welcome
+dashboard, reporting the data imported, skipped, failed and completed. The process
+can take many minutes if the organization is big, so be patient. You can manually
+refresh both charts to confirm that the process is progressing, and it is not stuck.
+If you believe that there can be any problem during the process, the better way
+to find what is happening is:
+
+- **[check the components status](#how-can-i-see-the-status-of-source-d-ce-components)
+with `sourced status`**; `gitcollector` and `ghsync` should be `Up` (the process
+didn't finish yet), or `Exit 0` (the process finished succesfully). They are
+independent components, so they can finish on different order depending on how
+many repositories or metadata is needed to process.
+
+- **[check the logs](#how-can-i-see-logs-of-the-running-components) of the failing component with `sourced logs [-f] {gitcollector,ghsync}`**
+to get more info about the errors found.
+
+
+## How Can I Restart One Scraper?
```

Comment:
```suggestion
## How can I restart one scraper?
```

Reply:
etc

Reply:
https://github.com/src-d/guide/blob/3c4642fe844e8a392f06afa7c0fbe7805833ccc0/engineering/documentation.md#arbitrary-choices-and-consistency